examples/tests/sdk_tests/std/array.w

```diff
@@ -22,32 +22,106 @@ test "at()" {
 }
 
 //-----------------------------------------------------------------------------
-// push()
+// push() & pop()
 
 let a = MutArray<str>["hello"];
 assert(a.length == 1);
 a.push("world");
 assert(a.length == 2);
 assert(a.at(0) == "hello");
 assert(a.at(1) == "world");
+a.pop();
+assert(a.length == 1);
+assert(a.at(0) == "hello");
 
-test "push()" {
+test "pushAndPop()" {
   let a = MutArray<str>["hello"];
   assert(a.length == 1);
   a.push("world");
   assert(a.length == 2);
   assert(a.at(0) == "hello");
   assert(a.at(1) == "world");
+  a.pop();
+  assert(a.length == 1);
+  assert(a.at(0) == "hello");
 }
 
 //-----------------------------------------------------------------------------
-// TODO: https://github.com/winglang/wing/issues/2785
-
 // concat()
+
+let b = MutArray<str>["hello"];
+assert(b.length == 1);
+let c = MutArray<str>["wing"];
+assert(c.length == 1);
+let d = b.concat(c);
+assert(d.length == 2);
+assert(d.at(0) == "hello");
+assert(d.at(1) == "wing");
+
+test "concat()" {
+  let b = MutArray<str>["hello"];
+  assert(b.length == 1);
+  let c = MutArray<str>["wing"];
+  assert(c.length == 1);
+  let d = b.concat(c);
+  assert(d.length == 2);
+  assert(d.at(0) == "hello");
+  assert(d.at(1) == "wing");
+}
+
+//-----------------------------------------------------------------------------
 // contains()
-// copy()
-// copyMut()
+
+let e = MutArray<str>["hello", "wing"];
+let f = "wing";
+let contains = e.contains(f);
+assert(contains);
+
+test "contains()" {
+  let e = MutArray<str>["hello", "wing"];
+  let f = "wing";
+  let contains = e.contains(f);
+  assert(contains);
+}
+
+//-----------------------------------------------------------------------------
 // indexOf()
+
+let g = MutArray<str>["hello", "wing"];
+let h = "wing";
+let index = g.indexOf(h);
+assert(index == 1);
+
+test "indexOf()" {
+  let g = MutArray<str>["hello", "wing"];
+  let h = "wing";
+  let index = g.indexOf(h);
+  assert(index == 1);
+}
+
+//-----------------------------------------------------------------------------
 // join()
+
+let i = MutArray<str>["hello", "wing"];
+let j = "wing";
+let delimeter = ";";
+let joinedString = i.join(delimeter);
+let expectedString = i.at(0) + delimeter + i.at(1);
+assert(joinedString == expectedString);
+
+test "join()" {
+  let i = MutArray<str>["hello", "wing"];
+  let j = "wing";
+  let delimeter = ";";
+  let joinedString = i.join(delimeter);
+  let expectedString = i.at(0) + delimeter + i.at(1);
+  assert(joinedString == expectedString);
+}
+
+//-----------------------------------------------------------------------------
+// TODO: https://github.com/winglang/wing/issues/2953
+// 
+// The bellow is not implemented:
+// copy()
```

Comment:
Let's add tests also for these 3 additional methods.

Reply:
Strange why you couldn't find them - where are you looking? They appear in the VSCode ext. completion and also in our [reference documentation](https://www.winglang.io/docs/standard-library/sdk#@winglang/sdk.std.ImmutableArray.lastIndexOf).
```ts
let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9];
arr.copyMut();

let mutArr = MutArray<num> [1, 2, 3, 4, 5, 6, 7, 8, 9];
mutArr.copy();

log("${arr.lastIndexOf(1)}");
```